src/modules/ekf2/EKF/ekf.h

```diff
@@ -540,6 +542,8 @@ class Ekf final : public EstimatorInterface
 	float _hagl_innov{0.0f};		///< innovation of the last height above terrain measurement (m)
 	float _hagl_innov_var{0.0f};		///< innovation variance for the last height above terrain measurement (m**2)
 
+	Vector3f _gravity_innov{};	///< innovation of the last gravity fusion observation (m/s**2)
```

Comment:
In current `main` we introduced a logging message per aid source that publishes for every sample update. 

https://github.com/PX4/PX4-Autopilot/blob/main/msg/EstimatorAidSource1d.msg

Search for `_aid_src_fake_hgt` in the EKF backend for a similar example. It publishes to the topic `estimator_aid_src_fake_hgt`.